assets/translations/en-us.yaml

```diff
@@ -5775,7 +5775,7 @@ typeLabel:
   management.cattle.io.setting: |-
     {count, plural,
       one { Advanced Setting }
```

Comment:
Shouldn't this also be changed to Settings ?

Reply:
hm I didn't catch this one since it's singular? When does it appear?